repo/packages/P/postgresql-admin/0/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "http://www.coretechnologies.com/images/postgresql-logo-48x48.png",
+    "icon-medium": "https://www.freebsdnews.com/wp-content/uploads/postgresql-96x96.png",
+    "icon-large": "https://www.iconattitude.com/icons/open_icon_library/apps/png/256/postgresql.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "postgresql-admin-docker": "zhajor/docker-phppgadmin:latest"
```

Comment:
only tag available. We could fork and apply tag -- let me know.
https://hub.docker.com/r/zhajor/docker-phppgadmin/builds/